student.rb

```diff
@@ -1,10 +1,10 @@
 require_relative 'person'
-# Student class inherits from Person
-class Student < Person
-  attr_reader :classroom
 
-  def initialize(*args, classroom)
-    super
+attr_reader :classroom
```

Comment:
- [ ] after the review from the previous CR you added this attribute reader however in order for this to work this line of code should be inside the `Student` class. below line 5 in this file

Reply:
Resolved 👍🏽